tests/runtime/builtin

```diff
@@ -157,7 +157,7 @@ EXPECT ERROR: failed to open file '/does/not/exist/file': No such file or direct
 
 NAME sizeof
 PROG struct Foo { int x; char c; } BEGIN { $x = 1; printf("%d %d %d %d %d\n", sizeof(struct Foo), sizeof((*(struct Foo*)0).x), sizeof((*(struct Foo*)0).c), sizeof(1 == 1), sizeof($x)); exit(); }
-EXPECT 8 4 1 1 8
```

Comment:
So the size of our boolean turned from 1 into 8? This by design?

Reply:
It went through a bit of a journey. In general, this is still a literal which is always treated as size 8. I can't remember exactly why this was changed during the fold literals pass, but this is the original behavior, so I wasn't worried: https://github.com/bpftrace/bpftrace/blob/86f98d9095e7c21f81c1dee17ce254592b0f8bad/tests/runtime/builtin#L160

(In other words, before this series and after this series it is still 8)